ROADMAP.md

```diff
@@ -9,6 +9,7 @@ Objective: "Simplify the user experience and provide a low barrier to entry by m
     * Deploy a ScikitLearn model without specifying a ScikitLearn Serving Technology.
     * Deploy a Pytorch model without specifying a Pytorch Serving Technology.
     * Deploy a Custom Containerized model by specifying your docker image and args.
+    * Deploy a model to run using TensorRT Serving Technology.
```

Comment:
Let's keep this our of the Core deliverables.

Reply:
Do you mean "out of the" aka delete this line?